src/ColorQueue.js

```diff
@@ -1,8 +1,28 @@
 import RNGUtil from './util/RNGUtil';
+import ColorUtil from './util/ColorUtil';
 
 export default class ColorQueue {
     constructor() {
         this.colors = ['#6E7DCE', '#F27E77', '#84E0A0', '#F2E796'];
```

Comment:
Minor: Could probably just replace all these entries with `hexToRgba(#color)` so we don't recalculate a bunch of times.